lib/instana/instrumentation/sidekiq-client.rb

```diff
@@ -4,6 +4,21 @@
 module Instana
   module Instrumentation
     class SidekiqClient
+      def self.redis_url
+        Sidekiq.redis_pool.with do |client|
+          host, port = 
+            case
+            when client.respond_to?(:config)
+              [client.config.host, client.config.port]
+            when client.respond_to?(:connection)
+              [client.connection[:host], client.connection[:host]]
```

Comment:
This line appear to assign the host to both, `host` and `port`.
Is it possible that you meant the following?
```suggestion
              [client.connection[:host], client.connection[:port]]
```